contracts/Example1.sol

```diff
@@ -2,6 +2,6 @@ pragma solidity ^0.8.4;
 
 contract Example1 {
     uint8 public day = 10;
-    uint256 public amount = 1000;
+    uint16 public amount = 1000;
     address public ownerAddr = 0x4e653b730367e303D30d8FB39d7350D07a60dfD6;
-}
\ No newline at end of file
+}
```

Comment:
The correct way of solving this is to move address above so that uint8 (day) and address (ownerAddr) are packed together